test_unstructured/partition/test_epub.py

```diff
@@ -4,22 +4,37 @@
 from unstructured.partition.epub import partition_epub
 
 DIRECTORY = pathlib.Path(__file__).parent.resolve()
+expected_sections = {
+    "CHAPTER I THE SUN-SEEKER",
```

Comment:
Are these actually out of order in the output or is this just because it's a set? Could you order these regardless so they're easier to read?

Reply:
Looks like the order is correct when I run the sample code, so just need to reorder these for readability.

Reply:
Sorry about that. It's just what I copied when I ran it as a set